packages/commonwealth/client/scripts/state/api/user/updateUser.ts

```diff
@@ -20,6 +19,20 @@ const useUpdateUserMutation = ({
 
   return trpc.user.updateUser.useMutation({
     onSuccess: (updated) => {
+      const keys = [
+        [ApiEndpoints.FETCH_PROFILES_BY_ID, undefined],
+        [ApiEndpoints.FETCH_PROFILES_BY_ID, user.id],
+      ];
+      keys.map((key) => {
+        queryClient.cancelQueries(key).catch(console.error);
+        queryClient.refetchQueries(key).catch(console.error);
+      });
+
+      updated.is_welcome_onboard_flow_complete &&
+        user.setData({
+          isWelcomeOnboardFlowComplete: true,
+        });
+
```

Comment:
lets remove this in favor of [this](https://github.com/hicommonwealth/commonwealth/pull/8574/commits/f165ef404234690ec8c12a4e4835091b56e51fac#r1693378902)

Reply:
the api call fails if the data is from another user

Reply:
@mzparacha leaving this last conversation open for your final review

Reply:
sry mb, i just realized i was mixing this with older logic that checked for profile on addresses.